<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqEditMenuBuilder</class>
 <widget class="QMenu" name="pqEditMenuBuilder">
  <property name="title">
   <string>&amp;Правка</string>
  </property>
  <action name="actionEditUndo">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqUndo24.png</normaloff>:/pqWidgets/Icons/pqUndo24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Отменить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionEditRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqRedo24.png</normaloff>:/pqWidgets/Icons/pqRedo24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Повторить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionEditCameraUndo">
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqUndoCamera24.png</normaloff>:/pqWidgets/Icons/pqUndoCamera24.png</iconset>
   </property>
   <property name="text">
    <string>Отменить действие камеры</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionEditCameraRedo">
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqRedoCamera24.png</normaloff>:/pqWidgets/Icons/pqRedoCamera24.png</iconset>
   </property>
   <property name="text">
    <string>Повторить действие камеры</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionChangeInput">
   <property name="text">
    <string>Изменить &amp;входные данные...</string>
   </property>
   <property name="iconText">
    <string>Изменить входные данные...</string>
   </property>
   <property name="toolTip">
    <string>Изменить входные данные фильтра</string>
   </property>
   <property name="statusTip">
    <string>Изменить входные данные фильтра</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqCopy22.png</normaloff>:/pqWidgets/Icons/pqCopy22.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Копировать</string>
   </property>
   <property name="statusTip">
    <string>Копировать свойства</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqPaste22.png</normaloff>:/pqWidgets/Icons/pqPaste22.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Вставить</string>
   </property>
   <property name="statusTip">
    <string>Вставить скопированные свойства</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../../../Widgets/Resources/QtWidgets.qrc">
     <normaloff>:/QtWidgets/Icons/pqDelete24.png</normaloff>:/QtWidgets/Icons/pqDelete24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Удалить</string>
   </property>
   <property name="statusTip">
    <string>Удалить</string>
   </property>
  </action>
  <action name="actionDelete_All">
   <property name="text">
    <string>Восстановить сессию по-умолчанию</string>
   </property>
  </action>
  <action name="actionIgnoreTime">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Не учитывать время</string>
   </property>
   <property name="toolTip">
    <string>Исключить время данного объекта/фильтра из анимации</string>
   </property>
   <property name="statusTip">
    <string>Исключить время данного объекта/фильтра из анимации</string>
   </property>
   <property name="whatsThis">
    <string>Исключить время данного объекта/фильтра из анимации</string>
   </property>
  </action>
  <action name="actionEditSettings">
   <property name="text">
    <string>Настройки...</string>
   </property>
   <property name="iconText">
    <string>Настройки</string>
   </property>
   <property name="toolTip">
    <string>Настройки</string>
   </property>
  </action>
  <action name="actionQuery">
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqFindData32.png</normaloff>:/pqWidgets/Icons/pqFindData32.png</iconset>
   </property>
   <property name="text">
    <string>Поиск данных...</string>
   </property>
   <property name="iconText">
    <string>Поиск данных</string>
   </property>
   <property name="toolTip">
    <string>Поиск данных по критерию текущего объекта (v).</string>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </action>
  <addaction name="actionEditUndo"/>
  <addaction name="actionEditRedo"/>
  <addaction name="separator"/>
  <addaction name="actionEditCameraUndo"/>
  <addaction name="actionEditCameraRedo"/>
  <addaction name="separator"/>
  <addaction name="actionQuery"/>
  <addaction name="actionChangeInput"/>
  <addaction name="actionCopy"/>
  <addaction name="actionPaste"/>
  <addaction name="actionIgnoreTime"/>
  <addaction name="actionDelete"/>
  <addaction name="actionDelete_All"/>
  <addaction name="separator"/>
  <addaction name="actionEditSettings"/>
 </widget>
 <resources>
  <include location="../../../Components/Resources/pqComponents.qrc"/>
  <include location="../../../Widgets/Resources/QtWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
